src/materials/Material.js

```diff
@@ -71,7 +71,6 @@ function Material() {
 	this.visible = true;
 
 	this.toneMapped = true;
-
```

Comment:
Leave `Material.js` alone 😁

Reply:
That new line totally fits there! :P I'll remove 👼